CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ separate changelogs for each crate were used. If you need to refer to these old
 
 ### Changed
 
+- libcnb.rs now targets [Buildpack API 0.9](https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.9). Buildpacks need to upgrade the `api` key to `0.9` in their `buildpack.toml`. ([#567](https://github.com/heroku/libcnb.rs/pull/567))
+  - `Process` no longer supports the `direct` flag. All processes are now `direct`. ([#567](https://github.com/heroku/libcnb.rs/pull/567))
```

Comment:
I wonder if it might help people migrating, if we mentioned something like ~"Processes that need to use bash should now wrap any commands with it explicitly"

Reply:
Nice idea, added something in 2f4ffb5.